frappe/core/doctype/doctype/doctype.json

```diff
@@ -22,6 +22,7 @@
   "track_views",
   "custom",
   "beta",
+  "allow_workflow",
```

Comment:
You need to add this option in Customize Form too

Reply:
Core doctypes cant be customized na:

![image](https://user-images.githubusercontent.com/15175501/86228288-d1177a80-bbab-11ea-9b6c-bb6f3a924224.png)